aopy/analysis.py

```diff
@@ -147,10 +159,11 @@ def run_curvefitting(means, make_plot=True, fig_title='Tuning Curve', n_subplot_
         n_cols (int) : No. of cols in subplot
 
     Returns:
-        tuple: Tuple containing:
-            | **params_day (Numpy array):** Curve fitting parameters
-            | **modulation depth (Numpy array):** Modulation depth of neuron
-            | **preferred direction (Numpy array):** preferred direction of neurons
+        params_day (Numpy array) : Curve fitting parameters
+        modulation depth (Numpy array) : Modulation depth of neuron
+        preferred direction (Numpy array) : preferred direction of neurons
```

Comment:
you should keep the `| **` stuff, that helps with formatting on the website. 

also, can you please use more descriptive types than "numpy array"? what is the size of each array (n_days, n_neurons, nch, etc.)